packages/host/config/environment.js

```diff
@@ -50,7 +53,6 @@ module.exports = function (environment) {
     // ENV.APP.LOG_TRANSITIONS = true;
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
-    ENV.featureFlags = {};
```

Comment:
This was making it so I couldn’t use SSE in development, it doesn’t seem correct to me to erase flags set via environment variables.